rules/falco_rules.yaml

```diff
@@ -2595,6 +2595,17 @@
   priority: CRITICAL
   tags: [process, mitre_execution]
 
+# This rule detects the execution of the docker client inside a container
+- list: k8s_client_binaries
+  items: [docker, kubectl, crictl]
+
+- rule: The docker client is executed in a container
+  desc: An event will trigger every time you run a docker client in a container
+  condition: spawned_process and container and proc.name in (k8s_client_binaries)
+  output: "Docker or kubernetes client in container (user=%user.name %container.info parent=%proc.pname cmdline=%proc.cmdline)"
```

Comment:
Minor: 
`ouptut: Docker or kubernetes client executed in container`

Reply:
Thanks Kaizhe for your support and help with my first PR in Falco... I'll commit this changes that you suggest, but before I'd like to fix the signature problem that it is giving in the this commit. 

I followed the link that you suggested and I added a gpg key (verified) and also a "Signed-off-by" in the commit. Even though, the problem persist... Any clue about how to fix it?

Reply:
can you try `rebase HEAD~3` and squash the one didn't commit?

Reply:
It seems it worked at last!  Thanks again for the support :)

Commit pushed with your suggestions.